crypto/encrypt_test.go

```diff
@@ -44,7 +44,7 @@ func TestTwofishEncryption(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if bytes.Compare(plaintext, decryptedPlaintext) != 0 {
+	if !bytes.Equal(plaintext, decryptedPlaintext) {
 		t.Fatal("Encrypted and decrypted zero plaintext do not match")
```

Comment:
the wallet has similar code. I'm wondering if this shouldn't just be moved to the crypto package.